tests/clone_test.go

```diff
@@ -0,0 +1,53 @@
+package tests_test
+
+import (
+	"flag"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	k8sv1 "k8s.io/api/core/v1"
+	k8smetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"kubevirt.io/containerized-data-importer/tests"
+)
+
+const (
+	TestNS = "default"
+)
+
+var _ = Describe("Cloning", func() {
+	var (
+		err error
+	)
+
+	flag.Parse()
+
+	BeforeEach(func() {
+		tests.CreatePVC(TestNS, "src", "1Mi")
+		Expect(err).To(BeNil())
+	})
+
+	AfterEach(func() {
+		tests.DeletePVC(TestNS, "src")
+	})
+
+	Specify("New source PVC should be empty", func() {
+		check := "[ $(ls -1 /pvc | wc -l) == 0 ]"
+		client, err := tests.GetKubeClient()
+		tests.PanicOnError(err)
```

Comment:
why aren't we using _Gomega_'s expected results pkg? The typical pattern is something like:
```
err := do-something()
Expect(err).ToBe(Nil), "extra informative err msg")
...
```